phpstan.neon

```diff
@@ -0,0 +1,17 @@
+parameters:
+    level: 8
+    checkMissingIterableValueType: false
+    excludes_analyse:
+
+    bootstrapFiles:
```

Comment:
Please, check if you need this

Reply:
need for future tests inside github actions

Reply:
Do you plan to add some acceptance tests?